config/proto/targets.proto

```diff
@@ -34,15 +47,17 @@ message TargetsDef {
 
 // REQUIRED, when using Ceph as the target system.
 // Comparable functionality is handled in the Go client library for GCS.
-message CephConnectionDetails {
-  int32 port = 1;  // Port for connecting to Ceph S3 API
-  
+message ConnectionDetails {
+  // REQUIRED for Ceph S3 API
```

Comment:
I don't think this message type is necessary: we could just as well define a stand-alon repeated field wherever it's needed.

Reply:
I am addressing comments on the .proto files in a different PR, as this branch is meant to only include the Cloudprober integration files. These .proto files are the same as those in the master branch.

Please see the new PR for the branch: evan/protoChanges.
---------------------------------------------------------------------------------------------------------------------------

I have removed this message and moved its contents into the `TargetDefinition` message.